extensions/default/src/getHangingProtocolModule.js

```diff
@@ -120,10 +120,10 @@ function getHangingProtocolModule() {
       name: hpMammography.id,
       protocol: hpMammography,
     },
-    // {
-    //   name: hpScale.id,
-    //   protocol: hpScale,
-    // },
+    {
+      name: hpScale.id,
+      protocol: hpScale,
+    },
```

Comment:
revert this for now since when you go to segmenttaion mode this gets matched for some reason

Reply:
Sure